src/halo/k8s/cluster-setup.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Rename to k8s_deploy.sh or deploy_k8s.sh. Try to follow a convention all across.

Reply:
done